Python-packages/covidcast-py/covidcast/geography.py

```diff
@@ -4,14 +4,19 @@
 from typing import Union, Iterable
 
 import pandas as pd
-import pkg_resources
-
-COUNTY_CENSUS = pd.read_csv(
-    pkg_resources.resource_filename(__name__, "geo_mappings/county_census.csv"), dtype=str)
-MSA_CENSUS = pd.read_csv(
-    pkg_resources.resource_filename(__name__, "geo_mappings/msa_census.csv"), dtype=str)
-STATE_CENSUS = pd.read_csv(
-    pkg_resources.resource_filename(__name__, "geo_mappings/state_census.csv"), dtype=str)
+import importlib_resources
+
+county_census = importlib_resources.files(__name__) / 'geo_mappings' / 'county_census.csv'
+with importlib_resources.as_file(county_census) as path:
```

Comment:
does this context manager afford us anything?  it looks like `pd.read_csv()` should be able to accept the `county_census` variable directly.